.github/workflows/coverage-report.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     - uses: actions/setup-java@v4
       with:
         distribution: 'temurin'
-        java-version: 8
+        java-version: 17
```

Comment:
Kafka Connect & Server are not available for Java < 17 anymore: https://kafka.apache.org/40/documentation/compatibility.html